system/Commands/Translation/LocalizationFinder.php

```diff
@@ -349,9 +367,13 @@ private function findLanguageKeysInFiles(array $files): array
 
             $this->writeIsVerbose('File found: ' . mb_substr($file->getRealPath(), mb_strlen(APPPATH)));
             $countFiles++;
-            $foundLanguageKeys = array_replace_recursive($this->findTranslationsInFile($file), $foundLanguageKeys);
+
+            $findInFile = $this->findTranslationsInFile($file);
+
+            $foundLanguageKeys[0] = array_replace_recursive($findInFile[0], $foundLanguageKeys[0]);
+            $foundLanguageKeys[1] = array_merge($findInFile[1], $foundLanguageKeys[1]);
```

Comment:
The variable names are difficult to understand. Don't use `[0]` or `[1]`.
Name meaningful variable names.

Reply:
Good. this is because the function returns multiple values instead of one